pkgs/tools/filesystems/btrfs-progs/default.nix

```diff
@@ -31,7 +33,11 @@ stdenv.mkDerivation rec {
     install -v -m 444 -D btrfs-completion $out/share/bash-completion/completions/btrfs
   '';
 
-  configureFlags = lib.optional stdenv.hostPlatform.isMusl "--disable-backtrace --disable-libudev";
+  configureFlags = let
+      musl = stdenv.hostPlatform.isMusl;
+    in
+      (lib.optional musl "--disable-backtrace") ++
+      (lib.optional (musl || disableUdev) "--disable-libudev");
```

Comment:
```suggestion
  configureFlags = lib.optional musl "--disable-backtrace"
   ++ lib.optional (stdenv.isMusl || disableUdev) "--disable-libudev";
```

Reply:
I've changed the structure as suggested but just to confirm, this should stay as `stdenv.hostPlatform.isMusl`, right?

Reply:
On is an alias to the other.